### ISSUE TITLE
Amp is referenced in partials/menu.blade used by non-amp base layout

### ISSUE THREAD

**Issue by pdbreen** [tipoff/support#132]
Multiple amp components are being pulled into the non-amp layout via the sharing of a partial:

`amp-lightbox`
https://github.com/tipoff/support/blob/f4185ab06edbdf634d4614ebaf2eb85b77fe696d/resources/views/partials/menu.blade.php#L1

`amp-accordian`
https://github.com/tipoff/support/blob/f4185ab06edbdf634d4614ebaf2eb85b77fe696d/resources/views/partials/menu.blade.php#L302

**Comment by drewroberts** [comment]
Yes, it is my preference to use those AMP elements in regular HTML pages. Just ignore it for right now and I'll make sure the elements get imported when we start working on those Layouts over the next few days.
